File: devops-scm-api/src/main/java/com/tencent/devops/scm/api/pojo/Status.java
package com.tencent.devops.scm.api.pojo;

import com.tencent.devops.scm.api.enums.StatusState;
import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
@Builder
public class Status {
    private StatusState state;
    private String context;
    private String desc;
    private String targetUrl;
}

File: devops-scm-api/src/main/kotlin/com/tencent/devops/scm/api/pojo/Status.kt
package com.tencent.devops.scm.api.pojo

import com.tencent.devops.scm.api.enums.StatusState

data class Status(
    // 状态值
    val state: StatusState,
    // 上下文标识
    val context: String,
    // 状态描述
    val desc: String,
    // 目标链接
    val targetUrl: String
)
